Makefile

```diff
@@ -9,24 +9,21 @@ endif
 
 ALL: $(CMDS)
 
-ui: ui/bindata.go
+ui: ui/.build/ui
 
 node_modules:
 	npm install
 
-$(GOPATH)/bin/houndd: ui/bindata.go $(SRCS)
+$(GOPATH)/bin/houndd: ui/.build/ui $(SRCS)
 	go install github.com/hound-search/hound/cmds/houndd
 
 $(GOPATH)/bin/hound: ui/bindata.go $(SRCS)
 	go install github.com/hound-search/hound/cmds/hound
 
-.build/bin/go-bindata:
-	GOPATH=`pwd`/.build go get github.com/go-bindata/go-bindata/...
-
-ui/bindata.go: .build/bin/go-bindata node_modules $(wildcard ui/assets/**/*)
-	rsync -r ui/assets/* .build/ui
+ui/.build/ui: node_modules $(wildcard ui/assets/**/*)
+	mkdir -p ui/.build
+	rsync -r ui/assets/* ui/.build/ui
 	npx webpack $(WEBPACK_ARGS)
-	$< -o $@ -pkg ui -prefix .build/ui -nomemcopy .build/ui/...
```

Comment:
Maybe this rule needs some tuning for proper rebuilds.

Reply:
Can you elaborate more on what you  mean here?

Reply:
I'll try. Since `ui/.build/ui` is a directory, I'm not sure under which circumstances make will figure out that the target needs to be rebuilt. Maybe it would be better to only check the JavaScript bundle generated by webpack?